### DISCUSSION TITLE
New Feature Proposal - User-Friendly Interface

### DISCUSSION THREAD

**Discussion by SimardeepSingh-zsh** [apache/streampipes#1874] [Ideas]
I'd like to propose a new feature that would greatly enhance the user experience of our software. Currently, our interface has several complexities that might deter new users. This feature would introduce a user-friendly mode, simplifying the interface for those who prefer a more streamlined experience. It would hide advanced settings and options by default, ensuring that users can focus on their tasks without distraction. Additionally, this mode could include interactive tooltips and a guided tour for first-time users. This enhancement aims to make our software more accessible to a broader audience and improve overall usability.

**Comment by tenthe** [comment]
Hi @SimardeepSingh-zsh ,

I completely agree, we should focus on a good first time user experience.
Since your suggestion is rather general, it would be very helpful if you could be more specific. 
What are the features of StreamPipes that made it difficult for you to get started with the software? 
Where exactly are tooltips missing that would be helpful?
Were there any tooltips that were difficult for you to understand?
What problems did you encounter when using the build in first time user guide?

Cheers, 
Philipp
